apps/meteor/app/api/server/helpers/getPaginationItems.ts

```diff
@@ -24,12 +28,14 @@ API.helperMethods.set('getPaginationItems', function _getPaginationItems(this: a
 		count = hardUpperLimit;
 	}
 
-	if (count === 0 && !settings.get('API_Allow_Infinite_Count')) {
+	const infiniteCount = (await Settings.findOneById('API_Allow_Infinite_Count'))?.value;
```

Comment:
any special reason for this to not be a `settings.get` as the settings above?

Reply:
Not really, I thought we should remove all settings.get in favor of the model, but let me undo the change.